rocq-runtime.opam.template

```diff
@@ -1,3 +1,11 @@
+depends: [
+  "dune" {>= "3.8"}
+  "ocaml" {>= "4.14.0"}
+  "ocamlfind" {>= "1.9.1" & (>= "1.9.8" | os != "windows") }
```

Comment:
Should be moved to dune-project

Reply:
Maybe we could move to 1.9.8, however as it is so recent we can maybe wait a bit.